### PR TITLE
Encode server error message

### PR DESCRIPTION
**Pull request by wapa5pow**
https://github.com/grpc/grpc-dart/issues/329

### CODE REVIEW

File: lib/src/server/handler.dart [hunk 1]
```diff
@@ -293,7 +293,13 @@ class ServerHandler_ extends ServiceCall {
     _customTrailers = null;
     outgoingTrailersMap['grpc-status'] = status.toString();
     if (message != null) {
-      outgoingTrailersMap['grpc-message'] = message;
+      try {
+        outgoingTrailersMap['grpc-message'] =
+            Uri.encodeFull(message).replaceAll("%20", " ");
+      } catch (_) {
```
Comment (sigurdm) [inline lib/src/server/handler.dart:299]: It is not clear to me what exceptions we can get here.

https://api.dart.dev/stable/2.8.4/dart-core/Uri/encodeFull.html doesn't mention any exceptions

Comment (wapa5pow) [reply lib/src/server/handler.dart]: You're right. Encoding string does not throw any exception. I fixed it.

**Comment by wapa5pow** [comment]
CI says EasyCLA expected but I already signed it before ( https://github.com/grpc/grpc-dart/pull/326#issuecomment-660697252 ). Maybe, because I force pushed my branch.

**Comment by sigurdm** [comment]
I'm trying to close and reopen.

**Pull request merged**